src/models/orderDetails.model.js

```diff
@@ -0,0 +1,15 @@
+"use strict";
+
+const createOrderDetailsTable = (sequelize, DataTypes) =>
```

Comment:
add curly braces "{}" and a return statement for the function,
or
add parenthesis "()" after the arrow function